src/styles/index.css

```diff
@@ -1,3 +1,424 @@
+  font-size: 0.875rem;
+  font-weight: 300;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  color: #4386bf;
+  text-align: left;
+  background-color: #fff;
+  cursor: pointer;
+}
+
+.book-menu-button:hover {
+  color: #0290ff;
+}
+
+.book-menu-item + .book-menu-item {
+  border-left: 1px solid #ccc;
+  padding-left: 20px;
+  background-color: #fff;
+}
+
+.book-progress-section {
+  display: flex;
+  background-color: #fff;
+  justify-content: center;
+  align-items: center;
+}
+
+.book-info-progress {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  padding-top: 30px;
+  background-color: #fff;
+}
+
+.level-progress {
+  width: 4.25rem;
+  height: 2.438rem;
+  margin: 0 5.75rem 0 0.688rem;
+  font-family: Montserrat;
+  font-size: 2rem;
+  font-weight: normal;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  color: #121212;
+  background-color: #fff;
+}
+
+.status-progress {
+  width: 5rem;
+  height: 1.125rem;
+  margin: 0 4.938rem 2.1rem 0.75rem;
+  opacity: 0.5;
+  font-family: Montserrat;
+  font-size: 0.875rem;
+  font-weight: normal;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  color: #121212;
+}
+
+.progress-img {
+  height: 90px;
+}
+
+.book-chapter-section {
+  display: flex;
+  flex-direction: column;
+  background-color: #fff;
+}
+
+.book-current-chapter {
+  width: 7.875rem;
+  height: 1.125rem;
+  margin: 0.063rem 3.625rem 0 3.688rem;
+  opacity: 0.5;
+  font-family: RobotoSlab;
+  font-size: 0.813rem;
+  font-weight: 300;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  color: #121212;
+}
+
+.book-title-chapter {
+  width: 4.688rem;
+  height: 1.313rem;
+  margin: 0.438rem 6.813rem 0.25rem 3.688rem;
+  font-family: RobotoSlab;
+  font-size: 1rem;
+  font-weight: 300;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: -0.4px;
+  color: #121212;
+  background-color: #fff;
+  padding-bottom: 15px;
+}
+
+.button-chapter {
+  width: 11.5rem;
+  height: 2.063rem;
+  margin: 0.75rem 0 0.563rem 3.688rem;
+  padding: 0.438rem 1.188rem 0.5rem 1.375rem;
+  border-radius: 3px;
+  background-color: #0290ff;
+  border: none;
+  color: #fafafa;
+  cursor: pointer;
+}
+
+.button-chapter:hover {
+  background-color: #46a8f3;
+}
+
+.line {
+  width: 1px;
+  height: 80px;
+  display: flex;
+  background-color: #a2a0a0;
+}
+
+#root {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.dropdown {
+  width: 17.813rem;
+  height: 31px;
+  margin-top: 19px;
+  padding-top: 14px;
+  padding-left: 15px;
+  border-radius: 4px;
+  border: solid 1px #a2a0a0;
+  font-family: 'Roboto', sans-serif;
+  font-size: 14px;
+  font-weight: normal;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  color: #747373;
 }
```

Comment:
- Your styles in not being applied to your app as you've imported in the index.js.. Kindly move all the styles to the index.css file in your root folder and delete the styles folder you've added to fix this issue.